src/main/webapp/app/entities/quiz-exercise/quiz-exercise-detail.component.ts

```diff
@@ -522,7 +570,9 @@ export class QuizExerciseDetailComponent implements OnInit, OnChanges, OnDestroy
         const areAllQuestionsValid = this.quizExercise.quizQuestions.every(function(question) {
             if (question.type === QuizQuestionType.MULTIPLE_CHOICE) {
                 const mcQuestion = question as MultipleChoiceQuestion;
-                return question.title && question.title !== '' && question.title.length < 250 && mcQuestion.answerOptions.some(answerOption => answerOption.isCorrect);
+                if (mcQuestion.answerOptions.some(answerOption => answerOption.isCorrect)) {
+                    return question.title && question.title !== '' && question.title.length < 250;
```

Comment:
question.title also checks that the String is non empty, so question.title !== '' is not needed.